### PR TITLE
4924 volume

### PR DESCRIPTION
**Pull request by tinybike**
Note: issue 4959 must be resolved (augur-core) and new contracts uploaded prior to merging this PR, due to backwards-incompatible changes to OrderCreated logs.

### CODE REVIEW

File: src/blockchain/log-processors/order-filled/calculate-number-of-shares-traded.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import BigNumber from "bignumber.js";
+
+export function calculateNumberOfSharesTraded(numShares: string, numTokens: string, price: string): string {
```
Comment (epheph) [inline src/blockchain/log-processors/order-filled/calculate-number-of-shares-traded.ts]: `Int256`? I know it's the same thing, but it seems like it's better documentation

File: src/blockchain/log-processors/order-created.ts [hunk 2]
```diff
@@ -37,55 +21,55 @@ interface OrderCreatedOnContractData {
...
-      parallel({
-        orderType: (next: AsyncCallback): void => augur.api.Orders.getOrderType(ordersPayload, next),
-        price: (next: AsyncCallback): void => augur.api.Orders.getPrice(ordersPayload, next),
-        amount: (next: AsyncCallback): void => augur.api.Orders.getAmount(ordersPayload, next),
-        sharesEscrowed: (next: AsyncCallback): void => augur.api.Orders.getOrderSharesEscrowed(ordersPayload, next),
-        moneyEscrowed: (next: AsyncCallback): void => augur.api.Orders.getOrderMoneyEscrowed(ordersPayload, next),
-      }, (err: Error|null, onContractData: OrderCreatedOnContractData): void => {
+      const fullPrecisionAmount = convertFixedPointToDecimal(amount, WEI_PER_ETHER);
+      const fullPrecisionPrice = denormalizePrice(minPrice, maxPrice, convertFixedPointToDecimal(price, numTicks));
+      if (new BigNumber(fullPrecisionPrice, 10).eq(new BigNumber(-10, 10))) process.exit();
```
Comment (epheph) [inline src/blockchain/log-processors/order-created.ts]: ?

File: src/blockchain/log-processors/order-filled/update-orders-and-positions.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { convertFixedPointToDecimal } from "../../../utils/convert-fixed-point-to-decimal";
+
+interface OrderFilledOnContractData {
+  amount: string;
+  creatorPositionInMarket: Array<string>;
+  fillerPositionInMarket: Array<string>;
+}
+
+export function updateOrdersAndPositions(db: Knex, augur: Augur, trx: Knex.Transaction, marketID: Address, shareToken: Address, orderID: Bytes32, creator: Address, filler: Address, numTicks: string|number, callback: ErrorCallback): void {
+  const shareTokenPayload = { tx: { to: shareToken } };
```
Comment (epheph) [inline src/blockchain/log-processors/order-filled/update-orders-and-positions.ts]: It looks like the argument `shareToken` and the `shareTokenPayload` object are not used?

File: src/blockchain/log-processors/order-created.ts (hunk 2 above)
Comment (tinybike) [reply src/blockchain/log-processors/order-created.ts]: Oops, I put that in as a debugging check and forgot to remove it... yikes :grimacing:

File: src/blockchain/log-processors/order-filled/update-orders-and-positions.ts (hunk 3 above)
Comment (tinybike) [reply src/blockchain/log-processors/order-filled/update-orders-and-positions.ts]: Good catch

**Comment by tinybike** [comment]
Think I'm gonna go ahead and merge this to avoid merge conflicts!

**Pull request merged**